reducer/src/main/java/com/graphicsfuzz/reducer/reductionopportunities/InterfaceBlockReductionOpportunities.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2021 The GraphicsFuzz Project Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.graphicsfuzz.reducer.reductionopportunities;
+
+import com.graphicsfuzz.common.ast.TranslationUnit;
+import com.graphicsfuzz.common.ast.decl.Declaration;
+import com.graphicsfuzz.common.ast.decl.InterfaceBlock;
+import com.graphicsfuzz.common.ast.expr.VariableIdentifierExpr;
+import com.graphicsfuzz.common.transformreduce.ShaderJob;
+import com.graphicsfuzz.common.typing.ScopeEntry;
+import com.graphicsfuzz.common.util.ListConcat;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class InterfaceBlockReductionOpportunities
+    extends ReductionOpportunitiesBase<InterfaceBlockReductionOpportunity> {
+
+  private Set<InterfaceBlock> referencedInterfaceBlocks;
```

Comment:
Nit at this stage in the project, and it is fairly obvious, but: Could have a comment on what the reduction opportunity is about, and also a comment on why this field is needed?

Reply:
Done.